packages/python/plotly/plotly/figure_factory/_dendrogram.py

```diff
@@ -25,14 +25,22 @@ def create_dendrogram(
     color_threshold=None,
 ):
     """
-    Function that returns a dendrogram Plotly figure object.
+    Function that returns a dendrogram Plotly figure object. This is a thin
+    wrapper around scipy.cluster.hierarchy.dendrogram.
 
     See also https://dash.plot.ly/dash-bio/clustergram.
 
     :param (ndarray) X: Matrix of observations as array of arrays
     :param (str) orientation: 'top', 'right', 'bottom', or 'left'
     :param (list) labels: List of axis category labels(observation labels)
-    :param (list) colorscale: Optional colorscale for dendrogram tree
+    :param (list) colorscale: Optional colorscale for the dendrogram tree. With
+                              scipy<=1.4.1 requires 8 colors to be specified,
+                              the 7th of which is ignored.  With scipy>=1.5.0,
+                              requires 10 colors. In this case the 8th color is
```

Comment:
so actually we only ever want 8 here, and the 7th is always ignored.

Reply:
right?

Reply:
Right what I wrote this time is actually wrong. Yes we always want 8, and the 7th is ignored. How about this:
```
:param (list) colorscale: Optional colorscale for the dendrogram tree.                          Requires 8 colors to be specified, the 7th of
                          which is ignored.  With scipy>=1.5.0, the 2nd, 3rd
                          and 6th are used twice as often as the others.
                          Given a shorter list, the missing values are
                          replaced with defaults and with a longer list the
                          extra values are ignored.
```

Reply:
Looks great! 💃  once done